_data/standing_meetings.yml

```diff
@@ -15,6 +15,10 @@
   time_cest: "14:00"
   connect: "[Link to zoom](https://princeton.zoom.us/j/97915651167?pwd=MXJ1T2lhc3Z5QWlYbUFnMTZYQlNRdz09)"
   agenda:
+    - title: "Add Initial Integration of Clad with Enzyme - Final Presentation"
+      date: 2022-08-31 14:00:00 +0200
+      speaker: "Manish Kausik H"
+      link: "[Slides](/assets/presentations/Manish_Integration_of_Enzyme_with_Clad_Final_Presentation.pdf)"
```

Comment:
Can you use this name: `CaaS_Weekly_20_07_22_Manish-Integration_of_Enzyme_with_Clad.pdf`